docs/tutorials/batch_showcase.ipynb

```diff
@@ -0,0 +1,578 @@
+{
```

Comment:
<p>I believe that we should prefer the following pattern for array building, because <code>np.append</code> allocates too much. I understand that practically in this case difference is small, however I would prefer to give best-practice code in the docs.</p><p> </p><pre class="ql-syntax" spellcheck="false">%%timeit
obj_ids = []
for i in range(10,110):
    obj_ids.append(np.array([i]*1250))
obj_ids = np.concatenate(obj_ids)
  
3.76 ms ± 18.7 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)
</pre><p>Compare to</p><pre class="ql-syntax" spellcheck="false">%%timeit
obj_ids = np.array([])
for i in range(10,110):
    obj_ids = np.append(obj_ids,np.array([i]*1250))
5.66 ms ± 114 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)  
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/327/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
implemented